docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -698,7 +693,8 @@ WITH (format='CSV',
     - Description
   * - ``format``
     - Optionally specifies the format of table data files; either ``PARQUET``,
-      ``ORC`, or ``AVRO``.  Defaults to ``ORC``.
+      ``ORC`, or ``AVRO``. Defaults to ``PARQUET`` from the default value
+      of ``iceberg.file-format`` or uses its setting from the catalog properties file.
```

Comment:
Do we need to tie it to the file_format metadata?

Reply:
> Do we need to tie it to the file_format metadata?

I'm assuming here that you are referring to `$files` metadata table's `file_format` property.
In this case, my answer would be _no_.